net/yggdrasil/Makefile

```diff
@@ -0,0 +1,112 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=yggdrasil
+PKG_VERSION:=0.3.5
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://codeload.github.com/yggdrasil-network/yggdrasil-go/tar.gz/v$(PKG_VERSION)?
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=2c69029adeb053ad049e90f1e4b7efa986094779868da77464d3c869984e861b
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-go-$(PKG_VERSION)
+
+PKG_LICENSE:=GPL-3.0
+PKG_MAINTAINER:=William Fleurant <meshnet@protonmail.com>
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/yggdrasil
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Routing and Redirection
+	TITLE:=Yggdrasil supports end-to-end encrypted IPv6 networks
+	URL:=https://yggdrasil-network.github.io/
+	DEPENDS:=$(GO_ARCH_DEPENDS) @IPV6 +kmod-tun +@(mips):KERNEL_MIPS_FPU_EMULATOR
+	PKGARCH:=all
```

Comment:
I don't think this is correct... This marks the package as architecture-independent.

Reply:
you're right -- we'll have to remove line +27

    PKGARCH (optional)

    By default, packages are built for the target architecture, and the 
    ipk files generated are tagged that way. This is normally correct, 
    for any compiled code, but if a package only contains scripts or 
    resources, marking it with PKGARCH:=all will make a single ipk 
    file that can be installed on any target architecture. (It will still be 
    compiled into bin/packages/arch/ however)